dockstore-client/src/main/java/io/dockstore/client/cli/nested/AbstractEntryClient.java

```diff
@@ -244,6 +274,14 @@ public boolean processEntryCommands(List<String> args, String activeCommand) thr
             case "test_parameter":
                 testParameter(args);
                 break;
+            case "wes":
```

Comment:
Is the plan to merge this to develop or keep it in a separate branch until it's ready?

If it gets merged to develop, then these command line options will be available, perhaps before they should be.

If it works today, then never mind.

Reply:
I think we can merge to develop, but with ample warnings that this is an beta-quality feature (due to the difficulty having it properly submit to WES implementations) both in the CLI help and on docs.dockstore.org.

Reply:
Should the warnings be implemented in a different branch?

Reply:
Yes, would probably be simpler, avoid merge conflicts with others by merging this work soon

Reply:
Could you create a ticket for this so it doesn't get forgotten? It would need to done before the 1.6 release.

Reply:
https://ucsc-cgl.atlassian.net/browse/DOCK-575